src/api/BoardAPI.ts

```diff
@@ -0,0 +1,54 @@
+const get = (valueObj) => valueObj?.v ?? "";
+
+const getName = (userData) => get(userData[2]);
+
+const getOrg = (userData) => {
+  if (get(userData[0]).toLowerCase() === "members at large") return "";
+  return get(userData[0]).toLowerCase();
+};
+
+const getTitle = (userData) => {
+  if (!get(userData[1]) || get(userData[0] === "-----")) return "";
+  return get(userData[1]);
+};
+
+const getSpecialization = (userData) => {
+  if (!get(userData[3]) || get(userData[3]) === "-----") return "";
```

Comment:
confused by these numbers / where they come from?

Reply:
for this section of code in general, i think to allow for more code maintainbility, it'd be good to explain the general process for these (all the `gets` here seem to have the same pattern), and to potentially include in that general comment an example of a line. Alot of these numbers seem completely arbitrary and this section in general seems too complicated to maintain without the needed context